BLUESPAWN-win-client/src/util/permissions/permissions.cpp

```diff
@@ -365,54 +615,57 @@ namespace Permissions{
 		std::vector<WCHAR> Domain(dwDomainLen);
 		std::vector<WCHAR> Name(dwNameLen);
 
-		if(!LookupAccountSid(nullptr, sdSID.GetUserSID(), Name.data(), &dwNameLen, Domain.data(), &dwDomainLen, &SIDType)){
+		if (!LookupAccountSid(nullptr, sdSID.GetUserSID(), Name.data(), &dwNameLen, Domain.data(), &dwDomainLen, &SIDType)) {
 			LOG_ERROR("Error getting user " << GetLastError());
 			bExists = false;
-		} else{
+		}
+		else {
 			wDomainName = std::wstring(Domain.data());
 			wName = std::wstring(Name.data());
-			if(SIDType == SidTypeWellKnownGroup){
+			if (SIDType == SidTypeWellKnownGroup) {
 				LOG_VERBOSE(2, "Group with name " << wName << " is a well known group.");
-			} else if(SIDType == SidTypeGroup || SIDType == SidTypeAlias){
+			}
+			else if (SIDType == SidTypeGroup || SIDType == SidTypeAlias) {
 				LOG_VERBOSE(2, "Group with name " << wName << " found.");
-			} else{
+			}
+			else {
 				LOG_VERBOSE(3, "Group with name " << wName << " does not exist.");
 				bExists = false;
 			}
 		}
 	}
 
-	std::optional<Owner> GetProcessOwner(){
+	std::optional<Owner> GetProcessOwner() {
 		HANDLE hToken;
-		if(!OpenProcessToken(GetCurrentProcess(), TOKEN_QUERY, &hToken)){
+		if (!OpenProcessToken(GetCurrentProcess(), TOKEN_QUERY, &hToken)) {
 			LOG_ERROR("Couldn't access process token. Error " << GetLastError());
 			return std::nullopt;
 		}
 		DWORD dwSize{ 0 };
 		GetTokenInformation(hToken, TokenOwner, nullptr, dwSize, &dwSize);
-		PTOKEN_OWNER owner = (PTOKEN_OWNER) GlobalAlloc(GPTR, dwSize);
+		PTOKEN_OWNER owner = (PTOKEN_OWNER)GlobalAlloc(GPTR, dwSize);
```

Comment:
reinterpret_cast is better style.  While we're at it, can we wrap this in an RAII type (generic_wrapper, smart pointer, allocation wrapper)